dms-service/src/main/java/de/muenchen/oss/swim/dms/domain/model/UseCase.java

```diff
@@ -67,15 +92,25 @@ public enum CooSource {
          * Target coo is extracted from separate metadata file.
          */
         METADATA_FILE,
+        /**
+         * Target coo is extracted from filename with regex.
+         * {@link UseCase#filenameCooPattern}
+         */
         FILENAME,
+        /**
+         * Target coo via static filename map.
+         * Searches for key matching filename and uses value as target coo.
+         * {@link UseCase#filenameToCoo}
+         */
+        FILENAME_MAP,
         /**
          * Target coo is statically configured.
          * {@link UseCase#targetCoo}
          */
         STATIC,
         /**
-         * Target is resolved via OU {@link DmsTarget#joboe}.
+         * Target is OU work queue of {@link UseCase#username}.
          */
-        OU_DEFAULT
+        OU_WORK_QUEUE
```

Comment:
What happens when `OU_WORK_QUEUE` is set together with `type=INBOX`? 
If not valid, please mention in JavaDoc.